packages/contentful--create-contentful-app/lib/index.js

```diff
@@ -67,7 +68,16 @@ async function initProject() {
 
     console.log(`Creating a Contentful app in ${chalk.bold(tildify(fullAppFolder))}.`);
 
-    await cloneTemplate('typescript', fullAppFolder);
+    program
+      .option('--javascript, -js')
+      .option('--typescript, -ts')
+      .action(async (opts) => {
+        console.log()
```

Comment:
🤔  why is that here?

Reply:
![](https://media3.giphy.com/media/3oriO5t2QB4IPKgxHi/giphy.gif?cid=790b76116f841af27eb17dfa9f322d4db97009583f1b0484&rid=giphy.gif&ct=g)